### PR TITLE
Fix build script `--report` option

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: scripts/build/README.md [hunk 1]
```diff
@@ -79,8 +79,6 @@ yarn build --report=all
 yarn build --report=stdout --report=text --report=html
 ```
 
-**`--report` equals to `--report=html`**
```
Comment (sosukesuzuki) [inline scripts/build/README.md:82]: I don't know how to make it happen with `node:utils.parsArgs`

Comment (fisker) [reply scripts/build/README.md:82]: It can't work anymore, you can use `--report=all`

**Review by fisker — APPROVED** [review]

File: scripts/build/build-javascript-module.js [hunk 2]
```diff
@@ -211,8 +211,8 @@ function getEsbuildOptions({ file, files, shouldCollectLicenses, cliOptions }) {
             output: (dependencies) => file.dependencies.push(...dependencies),
           },
         }),
-      cliOptions.reports &&
-        esbuildPluginVisualizer({ formats: cliOptions.reports }),
+      cliOptions.report &&
```
[the remark this answers is not in the document]
Comment (sosukesuzuki) [reply scripts/build/build-javascript-module.js:214]: I don't have opinion about this.

[the remark this answers is not in the document]
Comment (fisker) [reply scripts/build/build-javascript-module.js]: Edited via web ui 😄

**Pull request merged**